package.json

```
@@ -1,36 +1,57 @@
 {
-  "name": "@openapi-contrib/json-schema-to-openapi-schema",
-  "version": "0.0.0-development",
-  "description": "Converts a JSON Schema to OpenAPI Schema Object",
-  "main": "index.js",
-  "bin": {
-    "json-schema-to-openapi-schema": "bin/json-schema-to-openapi-schema.js"
-  },
-  "types": "index.d.ts",
-  "scripts": {
-    "test": "nyc --reporter=html --reporter=text mocha",
-    "coverage": "nyc report --reporter=text-lcov"
-  },
-  "repository": "github:openapi-contrib/json-schema-to-openapi-schema",
-  "author": "OpenAPI Contrib",
-  "license": "MIT",
-  "engines": {
-    "node": ">=10"
-  },
-  "devDependencies": {
-    "coveralls": "^3.1.1",
-    "mocha": "^10.0.0",
-    "nock": "^13.2.9",
-    "nyc": "^15.1.0",
-    "should": "^13.2.3"
-  },
-  "dependencies": {
-    "@cloudflare/json-schema-walker": "^0.1.1",
-    "@stoplight/json-ref-resolver": "^3.1.4",
-    "@stoplight/yaml": "^4.2.3",
-    "chalk": "^5.0.1",
-    "node-fetch": "^2.6.0",
-    "tslib": "^2.4.0",
-    "yargs": "^17.5.1"
-  }
+	"name": "browser-json-schema-to-openapi-schema",
+	"version": "0.1.4",
```

Can we change this back so autorelease will work.
```suggestion
	"name": "@openapi-contrib/json-schema-to-openapi-schema",
	"version": "0.0.0-development",
```